about.html

```diff
@@ -12,53 +12,87 @@
 </head>
 
 <body>
+  <div class="bg-div">
+    <img class="logo-img" src="img/lambda-black.png">
+    <nav>
+        <ul>
+            <li><a href="index.html">Home</a></li>
+            <li><a href="about.html">About</a></li>
+            <li><a href="#">Products</a></li>
+            <li><a href="#">Blog</a></li>
+            <li><a href="#">Contact</a></li>
+        </ul>
+    </nav>
+</div>
+
+<nav class="jumbo">
+  <img src="img/jumbo-about.png" alt="jumbo" class="center">
+      </nav>
+
     <div class="container about-page">
      
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-
-
+ 
+   
```

Comment:
The content of the about page is a great place to use semantic HTML `<section> and/or <article>` elements.